ClinGen/clincoded
# #313: Adds delete button to PubMed articles in GDMs with no evidence

**Steps to Test #313**

1. Begin curation of a GDM
2. Add a new PMID with the "Add New PMID" button on the left panel of the GDM
3. Expect to see a delete button appear in the center PMID panel next to the "PubMed" button
4. Click the delete button and expect the PMID to be removed and to be navigated to the first article in the GDM or none if there are no other articles in the GDM
5. Try adding evidence or article notes and expect the delete button to be hidden.
6. Delete all evidence and article notes and expect the delete button to be shown again.

Note: This branch is branched from #2025
